core/src/main/kotlin/org/evomaster/core/problem/graphQl/Data.kt

```diff
@@ -0,0 +1,11 @@
+class Data {
```

Comment:
all these classes dealing with the object representation from the schema, could be added in a sub-package, e.g called `.schema`

Reply:
done